pkg/scheduler/framework/interface.go

```diff
@@ -106,9 +113,10 @@ const (
 // the reasons should explain why.
 // NOTE: A nil Status is also considered as Success.
 type Status struct {
-	code    Code
-	reasons []string
-	err     error
+	code         Code
+	reasons      []string
+	err          error
+	failedPlugin string
```

Comment:
can we have a comment documenting the expectations around this field since it may not always be set.

Reply:
ok, will do.

Reply:
should we just name it plugin? Still, with the expectation that it might be empty for Successful status